types/clone-deep/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for clone-deep 4.0
+// Project: https://github.com/jonschlinkert/clone-deep
+// Definitions by: Tanfonto <https://github.com/tanfonto>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.7
+
+export interface InstanceClone<T> {
```

Comment:
The interface name is longer than the function definition, I think you can just inline that.